_data/people.yml

```diff
@@ -384,3 +392,29 @@
     active: true
     url: https://www.cs.princeton.edu/~gh0477/
     picture: /assets/img/people/ghuang.jpg
+
+  - first_name: Amelia
+    last_name: Dobis
+    nick: amelia.dobis
+    type: phd
+    active: true
+    url: https://dobios.github.io/
+    picture: https://dobios.github.io/res/ppic3.jpg
```

Comment:
Let's import the picture into the site's assets. It's one thing if a link becomes broken, but another if an image stops showing up because the external site changed.